### PR TITLE
如果文章内容大于64K，会导致CONTENT_TOO_BIG，增加判断，如果超限，删除“attrs”

### PR DESCRIPTION
**Pull request by QQ-War**

### CODE REVIEW

File: efb_mp_instantview_middleware/telegraph.py [hunk 1]
```diff
@@ -4,6 +4,17 @@
...
+def stripKey(dict, key):
+        if type(dict).__name__=="dict":
+                for ownkey in dict.copy():
+                        if ownkey==key:
+                                dict.__delitem__(key)
+                        else:
+                                stripKey(dict[ownkey], key)
+        elif type(dict).__name__=='list':
+                for n in range(len(dict)):
+                        stripKey(dict[n],key)
```
Comment (Rongronggg9) [inline efb_mp_instantview_middleware/telegraph.py:16]:
```suggestion
def stripKey(node, key):
    if isinstance(node, dict):
        for ownkey in tuple(node.keys()):
            if ownkey == key:
                del node[key]
            else:
                stripKey(node[ownkey], key)
    elif isinstance(node, list):
        for sub_node in node:
            stripKey(sub_node, key)
```

Are you a js developer?

Disclaimer: I cannot guarantee that my version should work. I am just telling you how to write pythonic code.

**Review by Rongronggg9 — COMMENTED** [review]
There's no reason you drop all `attrs` here. Some attributes are necessary (e.g. `src` for ``). Alternatively, you should **filter** `attrs` in function `tag2node`. Basically speaking, allowing `src` for `` and `<video>`, `href` for `<a>`, otherwise drop all attributes.

Ref: https://telegra.ph/api#NodeElement

https://github.com/ehForwarderBot/efb-mp-instantview-middleware/blob/8776c309745acf76470e136335dd9bad8c7bdc66/efb_mp_instantview_middleware/telegraph.py#L61

File: efb_mp_instantview_middleware/telegraph.py (hunk 1 above)
Comment (QQ-War) [reply efb_mp_instantview_middleware/telegraph.py:16]: Thanks for your suggestion, just learned.
I am just interested in it and kinds of Obsessive-compulsive disorder.

**Comment by catbaron0** [comment]
Thanks for your work!

But I think it does not solve the problem.
The reason for this issue is some of the mp articles is composed by some template tools, which generates lots of unnecessary nested nodes such as bellow:
```json
            {
                "tag": "p",
                "attrs": {},
                "children": [
                    {
                        "tag": "p",
                        "attrs": {},
                        "children": [
                            "毕竟，立陶宛这个国家"
                        ]
                    },
                    {
                        "tag": "p",
                        "attrs": {},
                        "children": [
                            {
                                "tag": "strong",
                                "attrs": {},
                                "children": [
                                    {
                                        "tag": "p",
                                        "attrs": {},
                                        "children": [
                                            "足足拥有280万人口，"
                                        ]
                                    }
                                ]
                            }
                        ]
                    },
                    {
                        "tag": "p",
                        "attrs": {},
                        "children": [
                            "不到。"
                        ]
                    }
                ]
            },

```

Note that most of the `atts` nodes are empty, so even if this solution MAY work for some specific pages 'cause it does shorten the content, the problem still exists for others. Thus I don't think removal of `attrs` is the correct way to solve this problem. I don't know the answer as well though.

Thanks again for your idea and code!
I'm going to close this PR but please feel free to open a new one if you have any other thoughts!

**Pull request closed without merging**